2022-01-17 17:46:55,093 INFO    Thread-2  :16978 [wandb_setup.py:_flush():71] setting env: {'entity': 'amazonproject', 'project': 'Amazon-Reviews', 'root_dir': '/Users/augustehlers/OneDrive - Danmarks Tekniske Universitet/Studiemappe/11. semester/MLOps/MLOps_Amazon_review_project', 'sweep_id': 'veqoakq9', 'run_id': 'ye2508oy', 'sweep_param_path': '/Users/augustehlers/OneDrive - Danmarks Tekniske Universitet/Studiemappe/11. semester/MLOps/MLOps_Amazon_review_project/wandb/sweep-veqoakq9/config-ye2508oy.yaml'}
2022-01-17 17:46:55,093 INFO    Thread-2  :16978 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-17 17:46:55,093 INFO    Thread-2  :16978 [wandb_init.py:_log_setup():371] Logging user logs to /Users/augustehlers/OneDrive - Danmarks Tekniske Universitet/Studiemappe/11. semester/MLOps/MLOps_Amazon_review_project/wandb/run-20220117_174655-ye2508oy/logs/debug.log
2022-01-17 17:46:55,093 INFO    Thread-2  :16978 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/augustehlers/OneDrive - Danmarks Tekniske Universitet/Studiemappe/11. semester/MLOps/MLOps_Amazon_review_project/wandb/run-20220117_174655-ye2508oy/logs/debug-internal.log
2022-01-17 17:46:55,094 INFO    Thread-2  :16978 [wandb_init.py:init():404] calling init triggers
2022-01-17 17:46:55,094 INFO    Thread-2  :16978 [wandb_init.py:init():409] wandb.init called with sweep_config: {'batch_size': 200, 'drop_out': 0.25, 'epochs': 5, 'lr': 0.001, 'optimizer': 'sgd'}
config: {}
2022-01-17 17:46:55,094 INFO    Thread-2  :16978 [wandb_init.py:init():460] starting backend
2022-01-17 17:46:55,094 INFO    Thread-2  :16978 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-01-17 17:46:55,107 INFO    Thread-2  :16978 [backend.py:ensure_launched():216] starting backend process...
2022-01-17 17:46:55,116 INFO    Thread-2  :16978 [backend.py:ensure_launched():221] started backend process with pid: 16986
2022-01-17 17:46:55,117 INFO    Thread-2  :16978 [wandb_init.py:init():469] backend started and connected
2022-01-17 17:46:55,120 INFO    Thread-2  :16978 [wandb_run.py:_config_callback():956] config_cb None None {'batch_size': 200, 'drop_out': 0.25, 'epochs': 5, 'lr': 0.001, 'optimizer': 'sgd'}
2022-01-17 17:46:55,123 INFO    Thread-2  :16978 [wandb_init.py:init():533] updated telemetry
2022-01-17 17:46:55,153 INFO    Thread-2  :16978 [wandb_init.py:init():563] communicating current version
2022-01-17 17:46:56,119 INFO    Thread-2  :16978 [wandb_init.py:init():568] got version response 
2022-01-17 17:46:56,119 INFO    Thread-2  :16978 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-17 17:46:56,798 INFO    Thread-2  :16978 [wandb_init.py:init():606] starting run threads in backend
2022-01-17 17:46:58,736 INFO    Thread-2  :16978 [wandb_run.py:_console_start():1810] atexit reg
2022-01-17 17:46:58,736 INFO    Thread-2  :16978 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-17 17:46:58,736 INFO    Thread-2  :16978 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-17 17:46:58,737 ERROR   Thread-2  :16978 [wandb_run.py:_redirect():1748] Failed to redirect.
Traceback (most recent call last):
  File "/Users/augustehlers/miniconda3/envs/MLOPs_project/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 1741, in _redirect
    out_redir.install()
  File "/Users/augustehlers/miniconda3/envs/MLOPs_project/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/Users/augustehlers/miniconda3/envs/MLOPs_project/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/Users/augustehlers/miniconda3/envs/MLOPs_project/lib/python3.9/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/Users/augustehlers/miniconda3/envs/MLOPs_project/lib/python3.9/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread of the main interpreter
2022-01-17 17:46:58,738 INFO    Thread-2  :16978 [wandb_init.py:init():633] run started, returning control to user process
2022-01-17 17:47:07,457 INFO    MainThread:16978 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-01-17 17:47:07,459 INFO    MainThread:16978 [wandb_run.py:_restore():1752] restore
